### PR TITLE
Full Map Timor Journey

### LINKED ISSUE #135: Full map
- User wants to see full map Timor-Leste

### PR DESCRIPTION
**Pull request by eraulo**
Closes #135 
If user wants to see full map. Use can click the full map to see full map.

### CODE REVIEW

File: map/templates/map/fullmapview.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                {% endif %}
+            {% endfor %}
+            function contentPopup(photo){
+                var viazen_id = photo.viazen_id.toString();
+                content = "<div class='container'><div class='row'><div class='col-md-5'><img src='"+ photo.photo +" "+"' width='70'></div><div class='col-md-7'><b>"+ photo.district+"</b>, "+ photo.subdistrict +", "+ photo.suco +"</br><a href='/detail/viazen/"+viazen_id+"/"+photo.photo_id+"'>Detail</a></div></div></div>"
+                return content
+            }
+        </script>
+    </body>
+</html>
```
Comment (marobo) [inline map/templates/map/fullmapview.html:45]: Why did you repeated the code?

**Comment by marobo** [comment]
Why did you add this page?

**Comment by marobo** [comment]
I think if you say to git for close issue: #135, the git automation will be not working.
Please just type `Closes #135 `

**Comment by eraulo** [comment]
> Why did you add this page?

If user wants to see full map. They can click in the full map to see full map.

File: map/templates/map/fullmapview.html (hunk 1 above)
Comment (eraulo) [reply map/templates/map/fullmapview.html:45]: > Why did you repeated the code?

I will make the source code  in one place

File: map/templates/map/mapview.html [hunk 2]
```diff
@@ -122,7 +122,7 @@ <h5><b>{{ via.title }}</b></h5>
     <script>
         const districts = {{ districts|safe }};
         var timormap = L.map('mapid').setView({{ points.DEFAULT_CENTER }}, {{ points.DEFAULT_ZOOM }});
-        L.tileLayer('{{ url_openstreetmap }}', { minZoom: 8, maxZoom: 18 }).addTo(timormap);
+        L.tileLayer('{{ url_openstreetmap }}', { minZoom: 8, maxZoom: 18 ,attribution: '<a href="{% url "fullmap" %}">{% trans "Full Map" %}</a> {% trans "Journey" %}'}).addTo(timormap);
```
Comment (PeteCoward) [inline map/templates/map/mapview.html:125]:
```suggestion
        L.tileLayer('{{ url_openstreetmap }}', { minZoom: 8, maxZoom: 18 ,attribution: '<a href="{% url "fullmap" %}">{% trans "Full Map " %}</a>'}).addTo(timormap);
```

**Comment by marobo** [comment]
I approved but please solve those conflicts and commit maun Peter suggestion!

**Pull request merged**